cmd/init.go

```diff
@@ -1,35 +1,88 @@
 package commands
 
 import (
+	"github.com/SAP/cloud-mta-build-tool/internal/exec"
 	"os"
 
 	"github.com/spf13/cobra"
 
+	"github.com/SAP/cloud-mta-build-tool/internal/artifacts"
 	"github.com/SAP/cloud-mta-build-tool/internal/tpl"
 )
 
+const (
+	makefile    = "Makefile.mta"
+	makefileTmp = "Makefile_tmp.mta"
+)
+
 // flags of init command
 var initCmdSrc string
 var initCmdTrg string
 var initCmdDesc string
+var initCmdName string
 var initCmdMode string
 
+// flags of build command
+var buildProjectCmdSrc string
+var buildProjectCmdTrg string
+var buildProjectCmdDesc string
+var buildProjectCmdMode string
+var buildProjectCmdMtar string
+var buildProjectCmdPlatform string
+var buildProjectCmdStrict bool
+
 // init flags of init command
 func init() {
-	initCmd.Flags().StringVarP(&initCmdSrc, "source", "s", "", "Provide MTA source")
-	initCmd.Flags().StringVarP(&initCmdTrg, "target", "t", "", "Provide MTA target")
-	initCmd.Flags().StringVarP(&initCmdDesc, "desc", "d", "", "Descriptor MTA - dev/dep")
-	initCmd.Flags().StringVarP(&initCmdMode, "mode", "m", "", "Mode of Makefile generation - default/verbose")
+	// set flags of init command
+	initCmd.Flags().StringVarP(&initCmdSrc, "source", "s", "",
+		"the path to the MTA project; the current path is set as the default")
+	initCmd.Flags().StringVarP(&initCmdTrg, "target", "t", "",
+		"the path to the MBT results folder; the current path is set as the default")
+	initCmd.Flags().StringVarP(&initCmdDesc, "desc", "d", "",
+		`the MTA descriptor; supported values: "dev" (development descriptor, default value) and "dep" (deployment descriptor)`)
+	initCmd.Flags().StringVarP(&initCmdName, "name", "n", "",
+		"the name of the Makefile; Makefile.mta is set as the default")
+	initCmd.Flags().StringVarP(&initCmdMode, "mode", "m", "",
+		"Mode of Makefile generation - default/verbose")
```

Comment:
`the mode of the Makefile generation; supported values: "default" and "verbose"`)